commons/zenoh-protocol/src/transport/open.rs

```diff
@@ -124,6 +121,13 @@ pub mod ext {
     /// # Compression extension
     /// Used to negotiate the use of compression on the link
     pub type Compression = zextunit!(0x6, false);
+
+    /// # Patch extension
+    /// Used to negotiate the patch version of the protocol
+    /// if not present (or 0), then protocol as released with 1.0.0
+    /// if >= 1, then fragmentation start/stop marker
+    pub type Patch = zextz64!(0x7, false);
```

Comment:
I think the patch extension is not needed in the `Open` message. Having it simply in the `Init` will suffice.